storage/client/queries/queries.go

```diff
@@ -179,6 +193,20 @@ const (
 		LIMIT $2::bigint
 		OFFSET $3::bigint`
 
+	DebondingDelegationsTo = `
+		-- delegatee_info is used to calculate the debonding escrow amount of the delegators in base units
+		WITH delegatee_info AS (
+			SELECT escrow_balance_debonding, escrow_total_shares_debonding
+			FROM chain.accounts
+			WHERE address = $1::text
+		)
+		SELECT delegator, shares, debond_end, delegatee_info.escrow_balance_debonding, delegatee_info.escrow_total_shares_debonding
+			FROM chain.debonding_delegations, delegatee_info
+			WHERE delegatee = $1::text
+		ORDER BY debond_end
```

Comment:
Since this is given in epochs, it's quite coarse and bound to yield sorting ties, so API responses might not be fully deterministic. Let's add a secondary ORDER BY, probably `shares DESC, delegator` (like in the active delegations query).